src/Components/ChatContainer/ChatContainer.test.js

```diff
@@ -14,7 +14,7 @@ import rootReducer from '../../reducers';
 
 const testStore = createStore(rootReducer);
 
-let socket = new SocketMock();
+let socket
```

Comment:
what is this doing? how does it function if not set to any value?

Reply:
So here I am declaring a variable and then later on in the before each I am assigning it